index.html

```diff
@@ -13,14 +13,14 @@
     </header>
 
   <main>
-    <h1>ID de Autorização e Motivos de Erro</h1>
+    <h1>ACH Transaction Errors</h1>
 
-  <label for="campoTextoJson">Insira o Código JSON:</label>
-  <textarea id="campoTextoJson" rows="20" cols="70"></textarea>
+  <label for="TextFieldJson">Insert JSON code:</label>
+  <textarea id="TextFieldJson" rows="20" cols="70"></textarea>
 
-  <button id="botaoExtrair">Extrair Erro de Autorização</button>
+  <button id="Submit">Submit</button>
 
-  <p id="resultado"></p>
+  <p id="Result"></p>
```

Comment:
Nomes de variáveis sempre precisam iniciar com letra minúscula.
Tudo o que está em "id", "for", "class" são variáveis.